packages/bpk-component-select/src/BpkSelect.js

```diff
@@ -15,25 +15,48 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/* @flow */
 
-import PropTypes from 'prop-types';
+import PropTypes, { type Node } from 'prop-types';
 import React from 'react';
 import { cssModules } from 'bpk-react-utils';
 
 import STYLES from './bpk-select.scss';
 
 const getClassName = cssModules(STYLES);
 
-const BpkSelect = props => {
+export type Props = {
+  id: string,
+  name: string,
+  value: string,
+  className: ?string,
+  docked: ?boolean,
```

Comment:
These `?boolean` types are not correct, they should be `boolean` because they are all filled in by React via the `defaultProps` before the function is called.